libs/product/src/drivers/magento/models/magento-product.ts

```diff
@@ -33,6 +38,7 @@ export interface MagentoProduct {
 			}
 		}
 	};
+	stock_status?: string;
```

Comment:
yea, it should. I had to make some changes to some tests to get it to work, because typescript complains about getting a string from a json object instead of the enum.

Reply:
oh it doesn't do that for me. It builds and passes the tests without the changes though, is that also what you see?